lib/daru/dataframe.rb

```diff
@@ -1407,7 +1407,10 @@ def only_numerics opts={}
 
     # Generate a summary of this DataFrame with ReportBuilder.
     def summary(method=:to_text)
-      ReportBuilder.new(no_title: true).add(self).send(method)
+      #uncomment this line to use ReportBuilder gem to generate data frame summary
+      #ReportBuilder.new(no_title: true).add(self).send(method)
+      #comment below line when using Reportbuilder gem
```

Comment:
There's no need for these comments or even then `g_summary` method. ReportBuilder should go.